statistician/requirements.txt

```diff
@@ -1,9 +1,8 @@
 --extra-index-url https://packages.dea.ga.gov.au/
 
-datacube[performance,s3]==1.8.5
-aiobotocore[awscli,boto3]==1.3.3
```

Comment:
Unpinning this is dangerous

Reply:
it is pined under constraints file

Reply:
Still I think it needs to be pinned here, due to how we re-create the constraints.

Reply:
done :)